Dockerfile

```diff
@@ -84,6 +84,12 @@ RUN \
     && echo 'Cleaning up installation files' >&2 && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# compile suid create_user binary
+COPY create_user.c /tmp/create_user.c
+RUN gcc -DHOMEDIR=\"/data/riotbuild\" -DUSERNAME=\"riotbuild\" /tmp/create_user.c -o /usr/local/bin/create_user \
+    && chown root:root /usr/local/bin/create_user \
+    && chmod 4750 /usr/local/bin/create_user
```

Comment:
>> +# compile suid create_user binary
>> +    && chmod 4750 /usr/local/bin/create_user
> 
> Does this actually work with no exec bit on others?
Yes, as the default uid for "docker run" is 0, so the active user
belongs to the correct group.

> Could you change this to symbolic permissions? It makes it a bit easier to understand IMO.

Really? :) I never know how to specify the more complex ones using
symbolic permission...